.travis.yml

```diff
@@ -0,0 +1,34 @@
+language: cpp
+sudo: false
+dist: trusty
+compiler:
+  - clang++
+
+matrix:
+  include:
+    os:
+      - linux
+    addons:
+      apt:
+        sources:
+          - llvm-toolchain-trusty-4.0
+        packages:
+          - clang-4.0
+          - lcov
+    env:
+      - MATRIX_EVAL="CC=clang-4.0 && CXX=clang++-4.0"
+
+branches:
+  only:
```

Comment:
why?

Reply:
The matrix is mostly for later so I can use different compilers for testing. Also potentially for Mac building. Right now, even basic build is failing because of broken dependency I've been fixing. Once that is fixed, will introduce more builds.